hazelcast/src/test/java/com/hazelcast/nio/ClassLoaderUtilTest.java

```diff
@@ -0,0 +1,19 @@
+package com.hazelcast.nio;
+
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class ClassLoaderUtilTest extends HazelcastTestSupport {
+
```

Comment:
You'll find quite a lot of those test classes from me, since they are mostly missing, while the other parts are touched by the production code or other test code itself.